core/Cargo.toml

```diff
@@ -66,10 +66,10 @@ itertools = "0.11.0"
 chrono = { version = "0.4", default-features = false, features = ["clock"] }
 chrono-tz = { version = "0.8" }
 paste = "1.0.14"
-datafusion-common = { git = "https://github.com/viirya/arrow-datafusion.git", rev = "111a940" }
-datafusion = { default-features = false, git = "https://github.com/viirya/arrow-datafusion.git", rev = "111a940", features = ["unicode_expressions"] }
-datafusion-functions = { git = "https://github.com/viirya/arrow-datafusion.git", rev = "111a940" }
-datafusion-physical-expr = { git = "https://github.com/viirya/arrow-datafusion.git", rev = "111a940", default-features = false, features = ["unicode_expressions"] }
+datafusion-common = { git = "https://github.com/viirya/arrow-datafusion.git", rev = "57b3be4" }
```

Comment:
hmm, should we wait until the DF PR is merged and use `https://github.com/apache/arrow-datafusion.git` instead of your personal fork?

Reply:
It is pointed to my fork of DataFusion not just for this. We uses the fork of arrow-rs to use a workaround for the Java Arrow bug. So we must use the fork too in our DataFusion crate, otherwise rust compiler will complain duplicated structs etc.

Reply:
I see. I missed https://github.com/apache/arrow-datafusion-comet/pull/239. It is OK as long as we don't stay in this state for too long 😂

Reply:
Yea, we will change back to official arrow-rs and DataFusion once we get new Java Arrow release.